public/js/services/capi.js

```diff
@@ -1,11 +1,9 @@
 import { pandaFetch } from './pandaFetch';
 import { uriEncodeParams, sanitiseQuery } from '../util/uriEncodeParams';
 
-export const searchTags = (searchText, type = null) => {
+export const searchTags = (searchText) => {
```

Comment:
It looks like the [`FormFieldTagPicker`](https://github.com/guardian/atom-workshop/blob/b9fc1760213034bd282f53ac84ab3ef94e6a7b4c/public/js/components/FormFields/FormFieldTagPicker.js#L47) is still using this function with two params. Changing the signature doesn't seem to have broken anything, but is it worth removing that second parameter for clarity?

Reply:
oh good catch. shoulda checked that

Reply:
updated!